Assets/Scripts/Scenes/InGame/Enemy/Trace/Feature/FeatureView.cs

```diff
@@ -13,17 +16,51 @@ public class FeatureView : MonoBehaviour
         private GameObject[] _stageInteracts;
         private GameObject[] nearStageObject = null;
         private GameObject interactTarget = null;
+        private GameObject _floor;
         AudioSource _audioSource;
         private Subject<Unit> _onDestroy = new Subject<Unit>();
         public IObservable<Unit> onDestroy { get => _onDestroy; }
+        public ReactiveProperty<GameObject> TemperatureTrace = new ReactiveProperty<GameObject>();
```

Comment:
```suggestion
        private ReactiveProperty<GameObject> TemperatureTrace = new ReactiveProperty<GameObject>();
        public IObservable<GameObject> OnTemperatureTrace { get => TemperatureTrace ; }
```
値を直接他のクラスから触れるのはまずいので、カプセル化してほしいです